runtime/attachments_test.go

```diff
@@ -170,10 +170,18 @@ func TestAccountAttachmentExportFailure(t *testing.T) {
 		import Test from 0x1
 		pub fun main(): &Test.A? { 
 			let r <- Test.makeRWithA()
-			let a = r[Test.A]
+			var a = r[Test.A]
+
+			// just to trick the checker
+			a = returnSameRef(a)
```

Comment:
What is the need for this now?

Reply:
Could you please add this explanation as a comment to the code?